tests/test_shadow_fork.py

```
@@ -72,29 +72,37 @@ def test_create_fails_if_already_active(self, tmp_path):
+            fork.create(base_branch="nonexistent")
+
+            assert fork.base_branch == "HEAD"
+            mock_resolve.assert_called_with("nonexistent")
 
     def test_create_warns_on_uncommitted_changes(self, tmp_path, caplog):
         """Test that create warns about uncommitted changes."""
         fork = ShadowFork(repo_root=tmp_path)
 
-        with patch.object(fork, "_run_git") as mock_git:
+        with patch.object(fork, "_run_git") as mock_git, \
+             patch.object(fork, "_resolve_base_ref", return_value="main") as mock_resolve:
```

Ruff (0.14.14)

[error] Local variable `mock_resolve` is assigned to but never used

Remove assignment to unused variable `mock_resolve`

(F841)